content/rest-api/builds.md

```diff
@@ -5,6 +5,8 @@ weight: 3
 
 APIs for managing builds are currently available for developers to preview. During the preview period, the API may change without advance notice.
 
+If you have multiple similar workflows for the same project, you might be interested in using <a href="https://blog.codemagic.io/dynamic-workflows-with-codemagic-api/" target="_blank" onclick="sendGtag('Link_in_docs_clicked','dynamic-workflows-with-codemagic-api')">Dynamic Workflows</a> to make your life easier.

```

Comment:
I propose to have something more formal instead of "make your life easier" as it will be mainly companies/agencies that will need this feature, so I'm suggesting to use this: "If you have multiple similar workflows for the same project, you can configure your workflows dynamically using API calls, read more about it [here]." Also, since this information is not essential to the content of the doc, I'd add it as a note at the end of the page.